_config.yml

```diff
@@ -45,8 +45,10 @@ email: contact@hackersatupc.org
 
 # Social media + community variables
 twitter: hackersupc
-fb: hackersatupc
+fb: hackersupc
 fb-community: hackersatupc
+slack: http://slack.hackersatupc.org
```

Comment:
Minor: Is not related with this PR, but some browsers (like iOS Safari) warns when entering a `http` (not `https`) site. Is hard to change it? @casassg maybe knows 😄

Reply:
Looks like the `https` link is not redirecting to the Slack join page. 😕

Reply:
Yep, right now it's only `http`, that's why I am saying it should be in another PR (maybe not in a PR but some _config_ changes 🤔), I would accept this PR, create the `https`s and then change the links ☺️